integ-test/src/test/java/org/opensearch/sql/sql/MultiMatchIT.java

```diff
@@ -61,4 +63,65 @@ public void verify_wildcard_test() throws IOException {
     var result = new JSONObject(executeQuery(query, "jdbc"));
     assertEquals(10, result.getInt("total"));
   }
+
+  @Test
+  public void test_multimatch_alternate_parameter_syntax() {
+    String query = "SELECT Tags FROM " + TEST_INDEX_BEER
+        + " WHERE multimatch('query'='taste', 'fields'='Tags')";
+    var result = new JSONObject(executeQuery(query, "jdbc"));
```

Comment:
Other tests have something like:
`JSONObject result = executeJDBCRequest(query);`
Is it better to use this instead for simplicity?

Reply:
Yes :thumbsup: